### DISCUSSION TITLE
In search bar automatically adding letter p in initial

### DISCUSSION THREAD

**Discussion by unaisshemim** [tailwindlabs/tailwindcss#11528] [Help]
**What version of Tailwind CSS are you using?**

For example: v2.0.4

**What build tool (or framework if it abstracts the build tool) are you using?**

For example: postcss-cli 8.3.1, Next.js 10.0.9, webpack 5.28.0

**What version of Node.js are you using?**

For example: v12.0.0

**What browser are you using?**

For example: Chrome, Safari, or N/A

**What operating system are you using?**

For example: macOS, Windows

**Reproduction URL**

A Tailwind Play link or public GitHub repo that includes a minimal reproduction of the bug. **Please do not link to your actual project**, what we need instead is a _minimal_ reproduction in a fresh project without any unnecessary code. This means it doesn't matter if your real project is private/confidential, since we want a link to a separate, isolated reproduction anyways.

A reproduction is **required** when filing an issue — any issue opened without a reproduction will be closed and you'll be asked to create a new issue that includes a reproduction. We're a small team and we can't keep up with the volume of issues we receive if we need to reproduce each issue from scratch ourselves.

**Describe your issue**

Describe the problem you're seeing, any important steps to reproduce and what behavior you expect instead.

**Comment by moinulmoin** [comment]
you mean the search bar in the tailwindcss website? I checked it now, it's perfectly working. Let me know if you are still facing this issue.

**Comment by RobinMalfait** [comment]
Hey! Can you elaborate a bit more about what you mean exactly?
